distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestCatalog.java

```diff
@@ -1204,6 +1204,92 @@ public void testNoEnforceValidityErrorsOrWarnings() throws Exception {
     }
 
     @Test
+    public void testQueryErrors() throws Exception {
+        getServiceManager().startFeature(true,
+                "catalog-plugin-metacard-validation",
```

Comment:
Yeah it totally says it's already started every time I run it, but I figured better safe than sorry in case another test stops the feature for whatever reason. If this isn't a real concern I'll remove it